README.md

```diff
@@ -1,7 +1,185 @@
-# Booster Release Scripts
+# Release Instructions
```

Comment:
Good idea to put into the README

Reply:
Unfortunately Git didn't detect the rename because I also did a lot of changes.